cmd/api/src/database/analysisrequest_test.go

```
@@ -49,3 +49,67 @@ func TestAnalysisRequest(t *testing.T) {
 	_, err = dbInst.GetAnalysisRequest(testCtx)
 	require.ErrorIs(t, err, database.ErrNotFound)
 }
+
+func TestAnalysisRequest_MergeAnalysisSteps(t *testing.T) {
+	var (
+		testCtx = context.Background()
+		dbInst  = integration.SetupDB(t)
```

golangci-lint (2.12.2)

[minor] SA1019: integration.SetupDB is deprecated: this suite of integration utils is deprecated and should be avoided Integration tests should be updated to reflect the latest standards. See commit https://github.com/SpecterOps/BloodHound/commit/a6cc43013fd769b97cc52cbc60b2314494054c9a#diff-e6bcb50873ade3cf33cef4e3e0ff566fb8ac1367b4ade36f4511bc2172a760e1 for implementation guidance. Additional detailed information can be found in Confluence. 

(staticcheck)